### DISCUSSION TITLE
PTVO firmware cc2530

### DISCUSSION THREAD

**Discussion by PetrP2B** [Koenkk/zigbee2mqtt#6447] [Devices]
Hi, I don't know what I'm doing wrong, but I'm using a few cc2530s with the latest FW and zigbee2mqtt and Home Assistant, unfortunately I can't configure more than 5 outputs, although they see 8 in HA, but they don't work from 6. Thank you for your help.

**Comment by PetrP2B** [comment]
Eventually I managed to run it, I generated FW several times and flashed in cc2530 and still reported as old FW in zigbee2mqtt. Now I have succeeded and I cannot find out where the mistake was.

**Reply by m0wlheld** [reply]
Btw: Did you manage to read internal temperature and source voltage from the „standard“ exposes? Mine are at temperature_l8 and voltage_l7, but not on voltage or temperature.

**Reply by PetrP2B** [reply]
I haven't tested this much, but in PTVO FW I set the internal temperature to I7 and the internal voltage to I8 and it worked.
